packages/styles/scss/utilities/_text-truncate.scss

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright IBM Corp. 2021
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+.#{$prefix}--text-truncate--end {
```

Comment:
This is a good example of a file where we have both CSS classes we like to emit along with potentially mixins, kind of like visually hidden too, I think.

Do you think it'd be worth offering both mixins and CSS classes? Trying to figure out when would it be a good time to offer one or the other (or both) 🤔